uploadNeuroDB/NeuroDB/ExitCodes.pm

```diff
@@ -0,0 +1,314 @@
+use strict;
+use warnings;
+
+package NeuroDB::ExitCodes;
+
+=pod
+
+=head1 NAME
+
+NeuroDB::ExitCodes -- Class regrouping all the exit codes used by the imaging
+insertion scripts
+
+=head1 SYNOPSIS
+
+  use NeuroDB::ExitCodes;
+
+  # testing if an argument was given to the script
+  if ( !$ARGV[0] ) {
+      print $Help;
+      print "$Usage\n\tERROR: Missing argument\n\n";
+      exit $NeuroDB::ExitCodes::MISSING_ARG;
+  }
+
+  # if script ran successfuly, exit with success exit code (a.k.a. 0)
+  exit $NeuroDB::ExitCodes::SUCCESS;
+
+
+=head1 DESCRIPTION
+
+This class lists all the exit codes used by the imaging insertion scripts.
+
+The exit codes are organized per script, together with a section that
+represents common failures to most scripts. For each script, exit codes are
+organized based on their use (validation failures, database related failures,
+file related failures, script execution failures, study related failures,
+input error checking and setting failures). Note that not all of the possible
+exit codes are used by each script, giving some room to add some later on if
+needed.
+
+Below is a list of the possible exit codes organized per script:
+
+1. Common exit codes to most insertion scripts (exit codes from 0 to 19, 0 =
+exit script with success status)
+
+2. Exit codes from batch_uploads_imageuploader (exit codes from 20 to 49)
```

Comment:
Is this intentional 20 to 49 or you meant 20-39?
I don;t care either way... but I have a feeling you intended 39 instead of 49.